src/main/java/com/SAS/ClientApp/Controllers/AddPoliciesController.java

```diff
@@ -0,0 +1,133 @@
+package com.SAS.ClientApp.Controllers;
+
+import javafx.scene.control.Alert;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextField;
+import javafx.scene.control.ToggleGroup;
+import javafx.scene.layout.AnchorPane;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.json.JSONObject;
+
+import java.io.IOException;
+
+public class AddPoliciesController {
+    public AnchorPane addPoliciesPane;
+    public TextField league_name;
+    public TextField season_year;
+    public Button add_policies_btn;
+    public ToggleGroup ranking;
+    public ToggleGroup rounds;
+    public ToggleGroup points;
+
+    public void init() {
+        add_policies_btn.setOnAction(event -> addPolicies());
+    }
+
+    private void addPolicies() {
+        //check that all fields are full
+
+        if (!(league_name.getText() == null || league_name.getText() == ""||season_year.getText() == null || season_year.getText() == "")) {
+            try {
+                Integer.parseInt(season_year.getText());
+                CloseableHttpClient httpClient = HttpClients.createDefault();
+                try {
+                    HttpPost request = new HttpPost(MainController.serverURL + "/league/definePolicies");
+                    JSONObject json = new JSONObject();
+                    json.put("username", MainController.username);
+                    json.put("leagueName", league_name.getText());
+                    json.put("seasonName", season_year.getText());
+                    if(ranking.getSelectedToggle()==null||rounds.getSelectedToggle()==null||points.getSelectedToggle()==null){
+                        Alert alert = new Alert(Alert.AlertType.ERROR);
+                        alert.setContentText("Please choose all the policies");
+                        alert.show();
+                        throw new Exception();
+                    }
+                    else {
+                        if(ranking.getSelectedToggle().toString().charAt(16)=='1'){
```

Comment:
I think you accidentally switch the cases - 1 and 2.

Reply:
it is a bit complicated, but i dont..
the backend function written by amit and the frontend written by me wasnt alligned with the same number, but i verifed that it worked.

Reply:
OK.